src/queries/query-keys.ts

```diff
@@ -1,6 +1,15 @@
-import { GetProductsParamType } from '@/types';
+import {
+  GetProductsParamType,
+  PatchUserOnboardingBody,
+  PostCheckNicknameBody,
+  PostUserPetBody,
+} from '@/types';
 
 export const QUERY_KEYS = {
   GET_CATEGORIES: () => ['GET_CATEGORIES'],
   GET_PRODUCTS: (params: GetProductsParamType) => ['GET_PRODUCTS', params],
+  PATCH_ONBOARDING: (body: PatchUserOnboardingBody) => ['PATCH_ONBOARDING', body],
+  POST_CHECKNICKNAME: (body: PostCheckNicknameBody) => ['POST_CHECKNICKNAME', body],
+  POST_PETPROFILE: (body: PostUserPetBody) => ['POST_PETPROFILE', body],
+  GET_USERNAME: () => ['GET_USERNAME'],
```

Comment:
동적인 값을 전달할 때에만 `() => {}` 형식으로 작성합니다! 여기에서는 넘겨주는 인자가 없으니 아래처럼 변경해 주세요
```suggestion
  GET_USERNAME: ['GET_USERNAME'],
```